src/rovr_common/launch/sensors/aruco_localization.launch

```diff
@@ -1,27 +1,24 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <launch>
-  <!-- args to be passed in -->
   <arg name="device" default="/dev/video0" />
   <arg name="show" default="true" />
```

Comment:
Let's get rid of these args. It might be nice instead to add arguments so that we can launch two aruco_localization nodes, one for each camera, just by adding an argument in the master launch file.